community/contributing/pr_workflow.rst

```diff
@@ -75,7 +82,9 @@ Godot repo, with your GitHub username as namespace:
 .. image:: img/github_fork_url.png
 
 You can then *clone* your fork, i.e. create a local copy of the online
-repository (in Git speak, the *origin remote*):
+repository (in Git speak, the *origin remote*). If you haven't already,
```

Comment:
Those instructions are Windows-specific (Linux users would typically get Git from their package manager, and use it directly in their default shell without having to run the bundled Bash that comes with Git on Windows). It's OK to mention it, but it would likely be better as a `.. note::`.